### DISCUSSION TITLE
Calculate distance of pores from nearest grain boundary

### DISCUSSION THREAD

**Discussion by Remsdel** [mtex-toolbox/mtex#1173] [Ask Anything]
Dear MTEX community,
I am looking for advice to tackle the following problem:
Assuming an EBDS map where pores have been identified and the (X,Y) coordinates of their geometric center is known (or alternatively all X,Y positions of the pore's boundary), I would like to compute the distance from their geometric center (or alternatively from the closest edge of the pore) to the the nearest grain boundary. The ultimate goal is to classify in an automated fashion the pores as inter-granular (on or near grain boundary) or intra-granular (removed from grain boundaries). Any ideas are welcome, thanks in advance.
Rémi D.

**Comment by kilir** [comment]
Hi,
I would think `isInclusion` should be able to distinguish intergranular for intragranular pores.

Cheers,
Rüdiger
